lib.php

```diff
@@ -28,6 +28,8 @@
 require_once($CFG->libdir.'/filelib.php');
 require_once($CFG->libdir.'/deprecatedlib.php');
 require_once($CFG->dirroot.'/user/selector/lib.php');
+// hsuforum actions lib
+require_once($CFG->dirroot.'/mod/hsuforum/hsuforum_actions_lib.php');
```

Comment:
Is this the file that runs on every page?

Reply:
the function that runs on every page has been removed, so I needed to pull this into hsuforums lib.php